### PR TITLE
Drivers

### PR DESCRIPTION
**Pull request by majiroy**
API and DB have been added 
migration and seed have been created

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,17 @@
   "version": "1.0.0",
   "description": "Glasgow Graduation Project - Food Drivers System",
   "main": "index.js",
-  "dependencies": {},
-  "devDependencies": {},
+  "dependencies": {
```
Comment (kabaros) [inline package.json]: @majiroy you don't . need these dependencies in top level package.json, they are inside server already

File: server/package.json [hunk 2]
```diff
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "start": "node app.js",
+    "run dev": "nodemon app.js",
     "test": "echo \"Error: no test specified\" && exit 0",
     "dev": "nodemon app.js",
     "migrate": "knex migrate:latest && knex seed:run",
-    "create-migration": "knex migrate:make $1",
-    "create-seed": "knex seed:make $1"
+    "create-migration": "knex migrate:make",
```
Comment (kabaros) [inline server/package.json]: please put this back as it was .. `$1` is correct, this is an issue with Windows that it doesn't work.

File: server/package.json [hunk 3]
```diff
@@ -20,6 +21,7 @@
     "express-handlebars": "^3.0.0",
     "knex": "^0.15.2",
     "pg": "^7.4.3",
+    "save": "^2.3.2",
```
Comment (kabaros) [inline server/package.json]: why did you install a package called `save`?

File: server/seeds/drivers.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex("drivers")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("drivers").insert([
+        { driver_id: 14, name: "Bad Khaled" },
```
Comment (kabaros) [inline server/seeds/drivers.js]: nice driver name :)

Comment (majiroy) [reply server/seeds/drivers.js]: he will be grateful

File: server/package.json (hunk 3 above)
Comment (majiroy) [reply server/package.json]: i dont know i think that installed by mistake i will have a look of my codes

File: server/package.json (hunk 2 above)
Comment (majiroy) [reply server/package.json]: so now mine is working without $, well what is the deference between with $1 or without ?

File: server/package.json (hunk 3 above)
Comment (majiroy) [reply server/package.json]: i found out why save package has been installed , actually i have put "install express" like this "npm install express -- save" there is a space after - so shall i reinstall the express ?

**Pull request merged**